### DISCUSSION TITLE
Update buildah ubuntu repos

### DISCUSSION THREAD

**Discussion by alexissellier** [containers/buildah#4188] [Q&A] [answered]
HI,
As the 1.27 version is out, would you update ubuntu (and other OS) repo, so the last version of buildah is available ?

**Comment by rhatdan** [comment, marked as answer]
We don't ship repos of any of our tools, we rely on distributions and community to package it.

@lsm5 Correct?

**Reply by lsm5** [reply]
@alexissellier if you're looking for the distro's official packages, then it's best you check with the distro's package maintainers. We don't control those.

But, we do have `unstable` builds of the latest versions if you wanna try them, see: https://github.com/containers/podman/issues/14302#issuecomment-1192617223 . Buildah will use the same installation steps as podman.

**Reply by lsm5** [reply]
fwiw, I'm checking with the github actions people to use this `unstable` repo. See: https://github.com/actions/runner-images/issues/5578 . So, if things work there, there should be some assurance that it works well.

**Comment by alexdmiller** [comment]
Would you consider releasing binaries via GitHub, or offering instructions for installing up-to-date packages via an alternative route?

As it stands, the version in the official package managers is very out of date with the buildah documentation, and the instructions for building from source are either incorrect or incomplete (been trying to build for about 2 hours on Ubuntu 22.04 without success). I'd love to try buildah, but it's basically impossible to do so at the moment.
